src/compiler/parsing/watf/number-literals.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+
+export function parse32F( sourceString: string ): number {
+  return parseFloat(sourceString);
+}
+
+export function parse64F( sourceString: string ): number {
+  return parseFloat(sourceString);
+}
+
+export function parse32I( sourceString: string ): number {
+  let value = 0;
+  if (isHexLiteral(sourceString)) {
+    value = ~~parseInt(sourceString, 16);
+  } else if (isDecimalExponentLiteral(sourceString)) {
+
```

Comment:
Is this empty? Could we throw an error here, like: unsupported value...

Reply:
This is a TODO for literals of the form `234e-3` with decimal exponents. I'll throw an error here and will create issues for all cases that Im aware of that are missing 👍